experimental/PropertyDDS/services/property-query/tool/rest/get_mv.js

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+const yargs = require('yargs');
+const Fixtures = require('../../test/db/fixtures');
+
+global.targets = {
+  mhServerUrl: 'http://127.0.0.1:3070'
+};
+
+const processArgs = () => {
+  const argv = yargs.option('branchGuid', {
+    alias: 'b',
+    description: 'Branch guid',
+    type: 'string'
+  })
+  .option('commitGuid', {
+    alias: 'c',
+    description: 'Commit guid',
+    type: 'string'
+  })
+  .demandOption(['branchGuid', 'commitGuid']).argv;
+
+  return Promise.resolve(argv);
+};
+
+const doWork = async (a) => {
+  const headers = Fixtures.getRequestSignatureHeaders(a.branchGuid);
+  const result = await Fixtures.fetchMaterializedView(a.branchGuid, a.commitGuid, headers);
+  console.log(JSON.stringify(result, null, 2));
+};
+
+processArgs().then((a) => doWork(a));
```

Comment:
Hi @evaliyev - You're really close...  :-)

I think the last step is to split this PR into three steps.

1.  Commit changes in /experimental/ and publish to NPM.
2.  Commit changes in /server/ and publish to NPM.
3.  Commit changes to /tinylicious/.

The reason is that while developers can use '--symlink:full' locally to treat all packages as a single mega repo, our CI will build client, server, and tinylicious in isolation.  (The intention is to help avoid breakages, since client/server are deployed at different cadences.)